src/integrationTest/groovy/wooga/gradle/node/NodeReleasePluginCredentialsSpec.groovy

```diff
@@ -136,8 +150,31 @@ class NodeReleasePluginCredentialsSpec extends GithubIntegrationSpec {
         }
         """.stripIndent()
 
-        then: "runs"
-        runTasksSuccessfully("ensureNpmrc")
+        when:
+        def result = runTasksSuccessfully("ensureNpmrc")
+
+        then:
+        !result.wasSkipped(":ensureNpmrc")
+        result.wasExecuted(":ensureNpmrc")
+    }
+
+    def "skips task of type NpmCredentialsTask with no set credentials"() {
```

Comment:
I would rename this test to make a little bit more readable:
`skips task of type NpmCredentialsTask when no credentials set`